BIPs/BIP-212, 213.json

```diff
@@ -0,0 +1,70 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1678805794131,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.13.3",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x388f6d71a610f5f3466fa248f484a7e9731ba02e708a32549b0775713f00b85a"
+  },
+  "transactions": [
+    {
+      "to": "0x7869296Efd0a76872fEE62A058C8fBca5c1c826C",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address",
+            "name": "contractAddress",
+            "type": "address"
+          }
+        ],
+        "name": "allowlistAddress",
+        "payable": false
+      },
+      "contractInputsValues": {
```

Comment:
Timeless will manage veBAL in the Timeless DAO multisig at 0x9a8fee232dcf73060af348a1b62cdb0a19852d13 and proposes to whitelist the Timeless Governance Multisig to interact with veBAL.